### PR TITLE
Ensure logrotate image is also pre-downloaded

### PR DESCRIPTION
**Pull request by luis5tb**

### CODE REVIEW

**Review by jpodivin — APPROVED** [review]

**Comment by luis5tb** [comment]
still testing this as it is not working as expected

**Comment by luis5tb** [comment]
now it is working

**Review by rebtoor — CHANGES_REQUESTED** [review]
edpm_logrotate_crond is already in the download_cache: https://github.com/openstack-k8s-operators/edpm-ansible/pull/338

**Comment by luis5tb** [comment]
> edpm_logrotate_crond is already in the download_cache: #338

but that is only for the packages, not for the container image

**Comment by rebtoor** [comment]
> > edpm_logrotate_crond is already in the download_cache: #338
> 
> but that is only for the packages, not for the container image

Oh, sorry, you're right. I'll leave you a review just to change a bit in the download task. (I saw that you already solved the merge issue, thanks!)

File: roles/edpm_download_cache/tasks/container_images.yml [hunk 1]
```diff
@@ -76,3 +76,11 @@
   ansible.builtin.include_role:
     name: osp.edpm.edpm_neutron_dhcp
     tasks_from: download_cache.yml
+
+- name: Download images for edpm_logrotate_crond role
```
Comment (rebtoor) [inline roles/edpm_download_cache/tasks/container_images.yml:80]: We already call download_cache.yml during the download of the packages so this task, as it stands, would be redundant. Do you mind splitting the download_cache in two different task files so we can call them independently?

File: roles/edpm_logrotate_crond/tasks/download_cache.yml [hunk 2]
```diff
@@ -9,3 +9,14 @@
   until: edpm_logrotate_crond_cronie_package_download.rc == 0
   retries: "{{ edpm_logrotate_crond_download_retries }}"
   delay: "{{ edpm_logrotate_crond_download_delay }}"
+
+- name: Download needed container images
+  containers.podman.podman_image:
+    name: "{{ item }}"
+  loop:
```
Comment (rebtoor) [inline roles/edpm_logrotate_crond/tasks/download_cache.yml:16]: I think we can avoid to use a loop here since a single image is downloaded.

**Review by rebtoor — CHANGES_REQUESTED** [review]

**Review by rebtoor — APPROVED** [review]

**Pull request merged**